matlab_simulation/02-examples_practice/gyroCovEstimation/gyroCovEstimation.m

```diff
@@ -0,0 +1,90 @@
+%% A Matlab program to calculate the convariances from a Gyroscope Data.
+% It demonstrates the following aspects:
+% ----Contributions in Ver.2.0
+% -Vectorized the Gyro Data.
+% -More Compact Code.
+% -Removed repeatations
+% -Completed the Error Distribution Comparison at the bottom.
+% -Commented wherever possible
+% 
+% Author: Bismaya Sahoo
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Initializations
+clear all; close all; clc;
+RAD2DEG=180/pi;
+
+tmin = 10; % Put Start_time here in seconds
```

Comment:
The formatting in this document is not consistent. For example: notice the difference between line 14 and 16, there is differences in the whitespace around the "=" sign. Please refer to the MatLab style guide:
[https://sites.google.com/site/matlabstyleguidelines/layout-documentation/white-space#TOC-Surround-and-by-spaces](https://sites.google.com/site/matlabstyleguidelines/layout-documentation/white-space#TOC-Surround-and-by-spaces)